### ISSUE TITLE
Sphinx build not emitting HTML on Python 3.7?

### ISSUE THREAD

**Issue by cdeil** [astropy/astropy-helpers#415]
This is a continuation of https://github.com/gammapy/gammapy/issues/1931 .
@bsipocz suggested to open an issue in this tracker in https://github.com/gammapy/gammapy/issues/1931#issuecomment-439184705

The issue is that in Gammapy we still need to support Python 2.7 for users, but in the dev environment I switched to Python 3.7 and there running `python setup.py build_docs` doesn't emit HTML in `docs/_build/html`, even if it finishes and claims to do so.

Might be related to https://github.com/astropy/sphinx-automodapi/pull/51

**Comment by bsipocz** [comment]
@astrofrog - Could we just backport a few PRs from 3.1 to address this?

**Comment by astrofrog** [comment]
#413 should fix this, so yes might make sense to backport

**Comment by astrofrog** [comment]
@cdeil - actually just to check, is the issue that the build is appearing in ``docs/docs/_build/html``?

**Comment by cdeil** [comment]
>  is the issue that the build is appearing in docs/docs/_build/html

Yes it is!
I hadn't realised this before.

This is what I get:
```
(gammapy-dev-37) hfm-1804a:gammapy deil$ python setup.py build_docs

<snip/>

dumping object inventory... done
build succeeded, 58 warnings.

The HTML pages are in docs/_build/html.

(gammapy-dev-37) hfm-1804a:gammapy deil$ ls -1 docs/
Makefile
_build
_static
_templates
api
astro
background
catalog
changelog.rst
conf.py
cube
data
datasets
detect
development
docs
getting-started.rst
image
index.rst
install
irf
make.bat
maps
nitpick-exceptions
references.rst
references.txt
scripts
spectrum
stats
time
tutorials.rst
utils
(gammapy-dev-37) hfm-1804a:gammapy deil$ ls -1 docs/_build/
doctrees
plot_directive
```

**Comment by astrofrog** [comment]
This is fixed by #428
